src/GitcoinGovernor.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+// forgefmt: disable-start
+import { GovernorCountingSimple, Governor } from "openzeppelin-contracts/governance/extensions/GovernorCountingSimple.sol";
+import { GovernorVotesComp, ERC20VotesComp } from "openzeppelin-contracts/governance/extensions/GovernorVotesComp.sol";
+import { GovernorTimelockCompound, ICompoundTimelock } from "openzeppelin-contracts/governance/extensions/GovernorTimelockCompound.sol";
+import { GovernorSettings } from "openzeppelin-contracts/governance/extensions/GovernorSettings.sol";
+// forgefmt: disable-end
+
+/**
+ * @notice The upgraded Gitcoin Governor—Bravo compatible and built from OpenZeppelin.
```

Comment:
I can't tell if this is correct and just sounds weird, or a typo haha. I think this is what you are going for:
```suggestion
 * @notice The upgraded Gitcoin Governor: Bravo compatible and built with OpenZeppelin.
```

The emdash renders like a hyphen which made it confusing

Reply:
done in 78a11b444add9c6e0fb59e6751995cb55d6c82dd